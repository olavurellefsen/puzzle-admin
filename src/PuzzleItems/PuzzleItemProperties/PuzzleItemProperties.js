import React from "react";
import {
  PuzzleItemPropertiesContainer,
  PuzzleItemPropertiesImage
} from "./PuzzleItemProperties.style";

export default () => (
  <PuzzleItemPropertiesContainer data-testid="PuzzleItemPropertiesContainer">
    <PuzzleItemPropertiesImage
      src={"https://via.placeholder.com/640x200?text=PuzzleItemProperties"}
      data-testid="PuzzleItemPropertiesImage"
    />
  </PuzzleItemPropertiesContainer>
);
