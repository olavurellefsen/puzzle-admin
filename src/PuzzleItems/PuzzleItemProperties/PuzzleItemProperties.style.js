import styled from 'styled-components';

export const PuzzleItemPropertiesContainer = styled.div`
  margin: 10px 10px;
`;

export const PuzzleItemPropertiesImage = styled.img`
`;
