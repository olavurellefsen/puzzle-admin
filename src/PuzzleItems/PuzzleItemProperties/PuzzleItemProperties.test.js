import React from 'react';
import { shallow } from 'enzyme';
import PuzzleItemProperties from './PuzzleItemProperties';

describe('PuzzleItemProperties', () => {
  let wrapper;
  beforeEach(() => wrapper = shallow(<PuzzleItemProperties />));

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());

  it('should render a PuzzleItemPropertiesContainer', () => {
    expect(wrapper.find('PuzzleItemPropertiesContainer').length).toEqual(1);
  });
});