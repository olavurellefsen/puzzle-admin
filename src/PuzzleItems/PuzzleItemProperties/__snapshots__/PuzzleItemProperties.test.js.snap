// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div
    class="sc-bdVaJa cyzjzt"
    data-testid="PuzzleItemPropertiesContainer"
  >
    <img
      class="sc-bwzfXH bFHdpU"
      data-testid="PuzzleItemPropertiesImage"
      src="https://via.placeholder.com/640x200?text=PuzzleItemProperties"
    />
  </div>
</div>
`;
